src/context/AllMedications.jsx

```diff
@@ -22,8 +22,7 @@ export const AllMedicationsContextProvider = (props) => {
       }));
     } catch (error) {
       setMedications((prevState) => ({
-        ...prevState,
-        isLoading: false,
+        isLoading: true,
```

Comment:
why we are making isLoading as true in catch block as when we call this method we make this as true initially and based on success and failure we set it as false

Reply:
this change was unintentional, changed it back to false